tests/validation/tests/v3_api/test_node_label.py

```diff
@@ -757,11 +759,13 @@ def test_node_label_custom_edit():
     node_labels[test_label] = new_value
     client.update(node, labels=node_labels)
     # cluster will go into updating state
+    print("before validate cluster state")
```

Comment:
we can remove this print statement here?

Reply:
Yes I'll will remove